Providers/Scripts/2.4x-2.5x/Scripts/nxOMSAgentNPMConfig.py

```diff
@@ -192,42 +215,44 @@ def ShowMof(op, ConfigType, ConfigID, Contents, Ensure, ContentChecksum):
 def Set(ConfigType, ConfigID, Contents, Ensure, ContentChecksum):
     ShowMof('SET', ConfigType, ConfigID, Contents, Ensure, ContentChecksum)
     retval = 0
-    if ConfigType != 'UpdatedAgentConfig':
```

Comment:
Why this check needs to be removed for the multi-homing? This is to ensure that schema contract is not violated